app/src/main/java/org/liberty/android/fantastischmemo/ui/RecentListFragment.java

```diff
@@ -411,15 +415,25 @@ public boolean onLongClick(View v) {
                 @Override
                 public void onClick(View v) {
```

Comment:
Whats going on in this method?

Reply:
when you click a deck on the recent list page it's checking if it's a multiple choice deck or a regular deck. And depending on the type it goes to a different a study activity